target/ppc/insn32.decode

```diff
@@ -367,7 +367,7 @@ DSCRIQ          111111 ..... ..... ...... 001100010 .   @Z22_tap_sh_rc
 
 ## Vector Bit Manipulation Instruction
 
-VGNB            000100 ..... -- ... ..... 10011001100   @VX_n
+VGNB            000100 ..... .. ... ..... 10011001100   @VX_n
```

Comment:
This change (replacing '-'s with '.'s) was supposed to be done in the format definition (VX_n), not here.

Reply:
Oops. Fixed in https://github.com/PPC64/qemu/pull/49/commits/7b4fa496310615bf5f7eda7b950fb96a186ff07b